config/peerpods/podvm/podvm-builder.sh

```diff
@@ -87,9 +86,28 @@ function create_podvm_image() {
 # Function to delete podvm image
 # IMAGE_ID or AMI_ID is the input and expected to be set
 # These are checked in individual cloud provider scripts and if not set, the script will exit
+# Accepts two optional arguments
+# -f : force delete the image
+# -g : delete the image gallery
```

Comment:
Please add `[-g]` to the usage at https://github.com/openshift/sandboxed-containers-operator/pull/404/commits/8caf648d06893619d282ae13b4e085cbe97f82a1#diff-6544cd9ec466fe5e11123ce3f835bf520a39dd0e245eb676ccff7bebda9c4757R233.